dev/drivers/scripts/stats/cam/jevs_cam_href_grid2obs_stats.sh

```diff
@@ -34,7 +34,7 @@ export run_mpi=yes
 export gather=yes
 
 
-export COMIN=/lfs/h2/emc/vpppg/noscrub/$USER/$NET/$evs_ver_2d
+export COMIN=/lfs/h2/emc/vpppg/noscrub/emc.vpppg/$NET/$evs_ver_2d
```

Comment:
revert lines 14 and 37 to original definitions.

Reply:
Done.